src/app/shared/components/display-table/metadata-table.component.html

```diff
@@ -99,10 +100,6 @@
         <div class="key">Message class name</div>
         <div class="value" [appCopyTooltip]="report.messageClassName">{{ report.messageClassName }}</div>
       </div>
-      <div class="table-col">
-        <div class="key">Path</div>
-        <div class="value" [appCopyTooltip]="report.path">{{ report.path }}</div>
-      </div>
```

Comment:
Does it have a special reason why this column is removed?

Reply:
yes, because neither reports or checkpoints have the `path` property